tests/untldoc_test.py

```diff
@@ -112,6 +112,33 @@ def test_untlpy2dict():
                                     'content': 'The Bronco'}]}
 
 
+@pytest.mark.parametrize('input_indent, json_output',
+                         [
+                             (4, ('{\n'
+                                  '    "title": [\n'
+                                  '        {\n'
+                                  '            "content": "The Bronco",\n'
+                                  '            "qualifier": "serialtitle"\n'
+                                  '        }\n'
+                                  '    ]\n'
+                                  '}')),
+                             (2, ('{\n'
```

Comment:
I think it would make sense to do `None` for the second indent arument if we are going with `parametrize` here, as I think that is the most compact version, which I would assume would be of more use to what is suggested in the issue.